ambry-api/src/main/java/com.github.ambry/account/Container.java

```diff
@@ -122,21 +122,64 @@
       "This is a container for the blobs without specifying a target account and container when they are put and isPrivate flag is true";
 
   /**
-   * The privacy setting of {@link #UNKNOWN_CONTAINER}.
+   * The encryption setting of {@link #UNKNOWN_CONTAINER}.
    */
-  public static final boolean UNKNOWN_CONTAINER_IS_PRIVATE_SETTING = false;
+  public static final boolean UNKNOWN_CONTAINER_ENCRYPTED_SETTING = false;
 
   /**
-   * The privacy setting for the containers to be associated with the blobs that are put without specifying a target
-   * container, but are specified private. {@link #DEFAULT_PRIVATE_CONTAINER} is one of the containers that use it.
+   * The encryption setting of {@link #DEFAULT_PUBLIC_CONTAINER}.
    */
-  public static final boolean DEFAULT_PUBLIC_CONTAINER_IS_PRIVATE_SETTING = false;
+  public static final boolean DEFAULT_PUBLIC_CONTAINER_ENCRYPTED_SETTING = false;
 
   /**
-   * The privacy setting for the containers to be associated with the blobs that are put without specifying a target
-   * container, but are specified public. {@link #DEFAULT_PUBLIC_CONTAINER} is one of the containers that use it.
+   * The encryption setting of {@link #DEFAULT_PRIVATE_CONTAINER}.
+   */
+  public static final boolean DEFAULT_PRIVATE_CONTAINER_ENCRYPTED_SETTING = false;
```

Comment:
Also wondering whether we need this big list of constants. Seems like we could save a lot of lines of code and make do with UNKNOWN_CONTAINER, DEFAULT_PUBLIC_CONTAINER, and DEFAULT_PRIVATE_CONTAINER

Reply:
Perhaps - let's remove them in a follow up?